lib/server/services/runDetector/getRunDetectorOrFail.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+const { getRunDetector } = require('./getRunDetector.js');
+const { NotFoundError } = require('../../errors/NotFoundError.js');
+const { utilities: { TransactionHelper } } = require('../../../database');
+
+/**
+ * Find an run detector model by its id and reject with a NotFoundError if none is found
+ *
+ * @param {number} runNumber the id of the run detector to find
```

Comment:
runNumber the id ? Maybe as you put it before `runNumber the identifier`

Reply:
done